contract-examples/contracts/ExampleDeployerList.sol

```diff
@@ -4,12 +4,148 @@ pragma solidity ^0.8.0;
 import "@openzeppelin/contracts/access/Ownable.sol";
 import "./IAllowList.sol";
 import "./AllowList.sol";
+import "ds-test/src/test.sol";
+
+address constant DEPLOYER_LIST = 0x0200000000000000000000000000000000000000;
 
 // ExampleDeployerList shows how ContractDeployerAllowList precompile can be used in a smart contract
 // All methods of [allowList] can be directly called. There are example calls as tasks in hardhat.config.ts file.
 contract ExampleDeployerList is AllowList {
   // Precompiled Allow List Contract Address
-  address constant DEPLOYER_LIST = 0x0200000000000000000000000000000000000000;
-
   constructor() AllowList(DEPLOYER_LIST) {}
+
+  function deployToken() public {
+    new AllowList(DEPLOYER_LIST);
```

Comment:
As per offline discussion with @ceyonur, should try using a different contract.

Reply:
Could be because of `IAllowList` intialization... something weird is going on there.

Reply:
created an issue here:
https://github.com/ava-labs/subnet-evm/issues/630

Reply:
Opened #630